app/src/main/java/ca/marshallasch/veil/services/VeilService.java

```diff
@@ -102,6 +112,11 @@ public void handleMessage(Message msg) {
                     }
 
                     rightMeshController.notifyNewContent(post, comment);
+                    if(post.getAnonymous()){
```

Comment:
This is in the wrong switch block, it will only send a notification to the device that created the post.